snmp/datadog_checks/snmp/config.py

```diff
@@ -393,13 +383,9 @@ def get_table_symbols(mib, table):
                 if 'symbol' in metric:
                     to_query = metric['symbol']
 
-                    try:
-                        _, parsed_metric_name = get_table_symbols(metric['MIB'], to_query)
-                    except Exception as e:
-                        warning("Can't generate MIB object for variable : %s\nException: %s", metric, e)
```

Comment:
This `warning()` line was last modified 9mo ago. The `get_table_symbols()` doesn't do any MIB resolution - it just builds `OID` objects and fills the `table_oids` list - which is why I'm assuming this is just outdated.